browser/chrome/browser/migration/migration.dtd

```diff
@@ -43,4 +44,4 @@
 <!ENTITY done.title                     "Importowanie zakończone">
 <!ENTITY done.label                     "Poniższe elementy zostały zaimportowane:">
 
-<!ENTITY closeSourceBrowser.label "Przed przejściem dalej, oroszę upewnić się, że wybrana przeglądarka jest wyłączona.">
+<!ENTITY closeSourceBrowser.label "Przed przejściem dalej proszę upewnić się, że wybrana przeglądarka jest wyłączona.">
```

Comment:
I don't know how to change a commit message for an already pushed commit, so I guess it's gonna be a lesson for the future. :)